azure-transfer-wf/main.nf

```diff
@@ -51,6 +51,7 @@ params.upload_cpus = null
 params.upload_mem = null
 params.upload_transport_mem = null
 
+params.enable_azure_songSub = false
```

Comment:
As this is more of an exception of a normal transfer process, I would make the logical reversed and name the param`params.skip_azure_songSub`